<html>
<head>
  <title>VEP summary</title>
  <script type="text/javascript" src="http://www.google.com/jsapi"></script>
  <script type="text/javascript">
    google.load('visualization', '1', {packages: ['corechart','table']});
  </script>
  <script type="text/javascript">
    
    function init() {
      // charts
      var var_class_pie = drawPie('var_class_pie', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['sequence_alteration',75],['insertion',1824],['deletion',1980],['SNV',47902]]), null);
var var_class_table = drawTable('var_class_table', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['sequence_alteration',75],['insertion',1824],['deletion',1980],['SNV',47902]]));

          google.visualization.events.addListener(var_class_pie, 'select', function() {
            var_class_table.setSelection(var_class_pie.getSelection());
          });
          google.visualization.events.addListener(var_class_table, 'select', function() {
            var_class_pie.setSelection(var_class_table.getSelection());
          });
        var var_cons_pie = drawPie('var_cons_pie', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',55],['splice_acceptor_variant',56],['stop_gained',119],['splice_donor_5th_base_variant',96],['frameshift_variant',209],['stop_lost',36],['start_lost',43],['inframe_insertion',117],['inframe_deletion',133],['protein_altering_variant',2],['missense_variant',10115],['splice_region_variant',612],['splice_polypyrimidine_tract_variant',2416],['splice_donor_region_variant',283],['incomplete_terminal_codon_variant',1],['synonymous_variant',10158],['stop_retained_variant',13],['coding_sequence_variant',2],['mature_miRNA_variant',28],['5_prime_UTR_variant',1571],['3_prime_UTR_variant',2213],['non_coding_transcript_exon_variant',5030],['intron_variant',17994],['upstream_gene_variant',28],['downstream_gene_variant',14],['intergenic_variant',437]]), {slices: [{color: "#ff7f50"}, {color: "#ff7f50"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {}, {color: "#ffd700"}, {color: "#ff7f50"}, {}, {}, {color: "#ff00ff"}, {color: "#76ee00"}, {color: "#76ee00"}, {color: "#458b00"}, {}, {}, {}, {}, {color: "#02599c"}, {color: "#a2b5cd"}, {color: "#a2b5cd"}, {color: "gray"}]});
var var_cons_table = drawTable('var_cons_table', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',55],['splice_acceptor_variant',56],['stop_gained',119],['splice_donor_5th_base_variant',96],['frameshift_variant',209],['stop_lost',36],['start_lost',43],['inframe_insertion',117],['inframe_deletion',133],['protein_altering_variant',2],['missense_variant',10115],['splice_region_variant',612],['splice_polypyrimidine_tract_variant',2416],['splice_donor_region_variant',283],['incomplete_terminal_codon_variant',1],['synonymous_variant',10158],['stop_retained_variant',13],['coding_sequence_variant',2],['mature_miRNA_variant',28],['5_prime_UTR_variant',1571],['3_prime_UTR_variant',2213],['non_coding_transcript_exon_variant',5030],['intron_variant',17994],['upstream_gene_variant',28],['downstream_gene_variant',14],['intergenic_variant',437]]));

          google.visualization.events.addListener(var_cons_pie, 'select', function() {
            var_cons_table.setSelection(var_cons_pie.getSelection());
          });
          google.visualization.events.addListener(var_cons_table, 'select', function() {
            var_cons_pie.setSelection(var_cons_table.getSelection());
          });
        var consequences_pie = drawPie('consequences_pie', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',265],['splice_acceptor_variant',322],['stop_gained',483],['splice_donor_5th_base_variant',1227],['frameshift_variant',827],['stop_lost',67],['start_lost',146],['inframe_insertion',881],['inframe_deletion',938],['protein_altering_variant',6],['missense_variant',79477],['splice_region_variant',20488],['splice_polypyrimidine_tract_variant',30625],['splice_donor_region_variant',3430],['incomplete_terminal_codon_variant',7],['synonymous_variant',99796],['start_retained_variant',4],['stop_retained_variant',95],['coding_sequence_variant',54],['mature_miRNA_variant',37],['5_prime_UTR_variant',13239],['3_prime_UTR_variant',23800],['non_coding_transcript_exon_variant',50311],['intron_variant',437095],['NMD_transcript_variant',43642],['non_coding_transcript_variant',85503],['upstream_gene_variant',914],['downstream_gene_variant',645]]), {slices: [{color: "#ff7f50"}, {color: "#ff7f50"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {}, {color: "#ffd700"}, {color: "#ff7f50"}, {}, {}, {color: "#ff00ff"}, {color: "#76ee00"}, {}, {color: "#76ee00"}, {color: "#458b00"}, {}, {}, {}, {}, {color: "#02599c"}, {}, {}, {color: "#a2b5cd"}, {color: "#a2b5cd"}]});
var consequences_table = drawTable('consequences_table', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',265],['splice_acceptor_variant',322],['stop_gained',483],['splice_donor_5th_base_variant',1227],['frameshift_variant',827],['stop_lost',67],['start_lost',146],['inframe_insertion',881],['inframe_deletion',938],['protein_altering_variant',6],['missense_variant',79477],['splice_region_variant',20488],['splice_polypyrimidine_tract_variant',30625],['splice_donor_region_variant',3430],['incomplete_terminal_codon_variant',7],['synonymous_variant',99796],['start_retained_variant',4],['stop_retained_variant',95],['coding_sequence_variant',54],['mature_miRNA_variant',37],['5_prime_UTR_variant',13239],['3_prime_UTR_variant',23800],['non_coding_transcript_exon_variant',50311],['intron_variant',437095],['NMD_transcript_variant',43642],['non_coding_transcript_variant',85503],['upstream_gene_variant',914],['downstream_gene_variant',645]]));

          google.visualization.events.addListener(consequences_pie, 'select', function() {
            consequences_table.setSelection(consequences_pie.getSelection());
          });
          google.visualization.events.addListener(consequences_table, 'select', function() {
            consequences_pie.setSelection(consequences_table.getSelection());
          });
        var coding_pie = drawPie('coding_pie', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['stop_gained',483],['frameshift_variant',827],['stop_lost',67],['start_lost',146],['inframe_insertion',881],['inframe_deletion',938],['protein_altering_variant',6],['missense_variant',79477],['incomplete_terminal_codon_variant',7],['synonymous_variant',99796],['start_retained_variant',4],['stop_retained_variant',95],['coding_sequence_variant',54]]), {slices: [{color: "#ff0000"}, {color: "#ff69b4"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {}, {color: "#ffd700"}, {color: "#ff00ff"}, {color: "#76ee00"}, {}, {color: "#76ee00"}, {color: "#458b00"}]});
var coding_table = drawTable('coding_table', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['stop_gained',483],['frameshift_variant',827],['stop_lost',67],['start_lost',146],['inframe_insertion',881],['inframe_deletion',938],['protein_altering_variant',6],['missense_variant',79477],['incomplete_terminal_codon_variant',7],['synonymous_variant',99796],['start_retained_variant',4],['stop_retained_variant',95],['coding_sequence_variant',54]]));

          google.visualization.events.addListener(coding_pie, 'select', function() {
            coding_table.setSelection(coding_pie.getSelection());
          });
          google.visualization.events.addListener(coding_table, 'select', function() {
            coding_pie.setSelection(coding_table.getSelection());
          });
        var chr_bar = drawBar('chr_bar', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['chr1',5041],['chr2',3451],['chr3',2609],['chr4',1846],['chr5',2114],['chr6',3109],['chr7',2403],['chr8',1729],['chr9',2049],['chr10',2181],['chr11',3289],['chr12',2678],['chr13',924],['chr14',2040],['chr15',1812],['chr16',2310],['chr17',3196],['chr18',823],['chr19',3769],['chr20',1360],['chr21',688],['chr22',1348],['chrX',1012]]), {legend: {position: "none"}});
var chr_table = drawTable('chr_table', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['chr1',5041],['chr2',3451],['chr3',2609],['chr4',1846],['chr5',2114],['chr6',3109],['chr7',2403],['chr8',1729],['chr9',2049],['chr10',2181],['chr11',3289],['chr12',2678],['chr13',924],['chr14',2040],['chr15',1812],['chr16',2310],['chr17',3196],['chr18',823],['chr19',3769],['chr20',1360],['chr21',688],['chr22',1348],['chrX',1012]]));

          google.visualization.events.addListener(chr_bar, 'select', function() {
            chr_table.setSelection(chr_bar.getSelection());
          });
          google.visualization.events.addListener(chr_table, 'select', function() {
            chr_bar.setSelection(chr_table.getSelection());
          });
        var protein_bar = drawBar('protein_bar', 'Position in protein', google.visualization.arrayToDataTable([['Position in protein (percentile)','Count'],['00-10%',17939],['10-20%',18113],['20-30%',18400],['30-40%',18218],['40-50%',17559],['50-60%',17934],['60-70%',18107],['70-80%',18518],['80-90%',18681],['90-100%',19206]]), {hAxis: {title: "Position in protein (percentile)", textStyle: {fontSize: 10}}, legend: {position: "none"}});

    }
    
    function drawPie(id, title, data, options) {    
      var pie = new google.visualization.PieChart(document.getElementById(id));
      pie.draw(data, options);
      return pie;
    }
    function drawBar(id, title, data, options) {
      var bar = new google.visualization.ColumnChart(document.getElementById(id));
      bar.draw(data, options);
      return bar;
    }
    function drawTable(id, title, data) {
      var table = new google.visualization.Table(document.getElementById(id));
      table.draw(data, null);
      return table;
    }
    function drawLine(id, title, data, options) {
      var line = new google.visualization.LineChart(document.getElementById(id));
      line.draw(data, options);
      return line;
    }
    function drawArea(id, title, data, options) {
      var area = new google.visualization.AreaChart(document.getElementById(id));
      area.draw(data, options);
      return area;
    }
    google.setOnLoadCallback(init);
  </script>
  
  
  <style type="text/css">
    body {
      font-family: arial, sans-serif;
      margin: 0px;
      padding: 0px;
    }
    
    a {color: #36b;}
    a.visited {color: #006;}
    
    .stats_table {
      margin: 5px;
    }
    
    td {
      padding: 5px;
    }

    th.gradient {
      height: auto;
    }

    .stats_table tr:nth-child(odd) {
      background-color: #f0f0f0;
    }
    
    h3 {
      color: #666;
    }
    
    .masthead {
      background-color: white;
      color: rgb(204, 221, 255);
      height: 80px;
      width: 100%;
      padding: 0px;
    }
    
    .main {
      padding: 10px;
    }
    
    .gradient {
      background: #333366; /* Old browsers */
      background: -moz-linear-gradient(left,  #333366 0%, #ffffff 100%); /* FF3.6+ */
      background: -webkit-gradient(linear, left top, right top, color-stop(0%,#333366), color-stop(100%,#ffffff)); /* Chrome,Safari4+ */
      background: -webkit-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Chrome10+,Safari5.1+ */
      background: -o-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Opera 11.10+ */
      background: -ms-linear-gradient(left,  #333366 0%,#ffffff 100%); /* IE10+ */
      background: linear-gradient(to right,  #333366 0%,#ffffff 100%); /* W3C */
      filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#333366', endColorstr='#ffffff',GradientType=1 ); /* IE6-9 */
      
      padding: 0px;
      height: 80px;
      width: 700px;
    }
    
    .main_content {
      margin-left: 300px;
    }
    
    .sidemenu {
      width: 260px;
      position: fixed;
      border-style: solid;
      border-width: 2px;
      border-color: rgb(51, 51, 102);
    }
    
    .sidemenu_head {
      width: 250px;
      background-color: rgb(51, 51, 102);
      color: rgb(204, 221, 255);
      padding: 5px;
    }
    
    .sidemenu_body {
      width: 250px;
      padding: 5px;
    }
  </style>
</head>
<body>
<div id="masthead" class="masthead">
  <div style="float: left; display: inline; padding: 10px; height: 80px;">
    <a href="http://www.ensembl.org/"><img src="http://static.ensembl.org/i/e-ensembl.png" width="185" height="45" alt="Ensembl"></a>
  </div>
  
  <div style="float: right; display: inline; height: 80px; background: white; padding: 10px;">
    <a href="http://www.ensembl.org/vep"><img src="http://www.ensembl.org/img/vep_logo.png" width="111" height="50" alt="VEP"></a>
  </div>
  <div class="gradient">
  </div>
</div>
<div class="main">
<div class="sidemenu"><div class="sidemenu_head">Links</div><div class="sidemenu_body"><ul><li><a href="#masthead">Top of page</a></li><li><a href="#run_stats">VEP run statistics</a></li><li><a href="#gen_stats">General statistics</a></li><li><a href="#var_class">Variant classes</a></li><li><a href="#var_cons">Consequences (most severe)</a></li><li><a href="#consequences">Consequences (all)</a></li><li><a href="#coding">Coding consequences</a></li><li><a href="#chr">Variants by chromosome</a></li><li><a href="#protein">Position in protein</a></li></ul></div></div><div class='main_content'><h3 id="run_stats">VEP run statistics</h3><table class="stats_table"><tr><td>VEP version (API)</td><td> 107 (107)</td></tr><tr><td>Annotation sources</td><td>Cache: /usr/local/bioinformatics/databases/vep/homo_sapiens_merged/107_GRCh38</td></tr><tr><td>Species</td><td>homo_sapiens</td></tr><tr><td>Command line options</td><td><pre>--af_1kg --af_gnomade --biotype --cache --canonical --dir /usr/local/bioinformatics/databases/vep --distance 10 --filter_common --force_overwrite --fork 12 --format vcf --full_command vep --input_file vcfs/JAS_N36.filt.vcf --output_file vcfs/JAS_N36.filt.vep.vcf --cache --merged --dir /usr/local/bioinformatics/databases/vep --fork 12 --force --format vcf --vcf --vcf_info_field ANN --canonical --no_intergenic --distance 10 --offline --use_given_ref --filter_common --mane_select --biotype --af_1kg --af_gnomade --input_file vcfs/JAS_N36.filt.vcf --mane_select --merged --no_intergenic --offline --output_file vcfs/JAS_N36.filt.vep.vcf --use_given_ref --vcf --vcf_info_field ANN</pre></td></tr><tr><td>Start time</td><td>2023-01-25 20:22:17</td></tr><tr><td>End time</td><td>2023-01-25 20:27:24</td></tr><tr><td>Run time</td><td>307 seconds</td></tr><tr><td>Input file</td><td>vcfs/JAS_N36.filt.vcf</td></tr><tr><td>Output file</td><td>vcfs/JAS_N36.filt.vep.vcf</td></tr></table><h3 id="gen_stats">General statistics</h3><table class="stats_table"><tr><td>Lines of input read</td><td>51781</td></tr><tr><td>Variants processed</td><td>51781</td></tr><tr><td>Variants filtered out</td><td>0</td></tr><tr><td>Novel / existing variants</td><td>0 (0.0) / 51781 (100.0)</td></tr><tr><td>Overlapped genes</td><td>32451</td></tr><tr><td>Overlapped transcripts</td><td>206919</td></tr><tr><td>Overlapped regulatory features</td><td>-</td></tr></table><hr/><h3 id="var_class">Variant classes</h3><div id="var_class_pie" style="width: 800px; height: 200px">&nbsp;</div><div id="var_class_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="var_cons">Consequences (most severe)</h3><div id="var_cons_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="var_cons_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="consequences">Consequences (all)</h3><div id="consequences_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="consequences_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="coding">Coding consequences</h3><div id="coding_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="coding_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr">Variants by chromosome</h3><div id="chr_bar" style="width: 800px; height: 200px">&nbsp;</div><div id="chr_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="protein">Position in protein</h3><div id="protein_bar" style="width: 800px; height: 200px">&nbsp;</div></div>
</div></body>
</html>
